compiler/crates/relay-typegen/tests/generate_typescript/fixtures/updatable-operation-assignable-fragment.expected

```diff
@@ -0,0 +1,27 @@
+==================================== INPUT ====================================
+query UpdatableQuery @updatable {
+  viewer {
+    ...Assignable_viewer
+  }
+}
+
+fragment Assignable_viewer on Viewer @assignable {
+  __typename
+}
+==================================== OUTPUT ===================================
+import { FragmentRefs } from "relay-runtime";
+export type UpdatableQuery$variables = {};
+export type UpdatableQuery$data = {
+  get viewer(): {} | null;
+  set viewer(value: {
+    readonly __typename: "Viewer";
+    readonly __id: string;
+    readonly " $fragmentSpreads": "Assignable_viewer";
```

Comment:
Should this be wrapped in a FragmentRefs type?

Reply:
I think this might've been a bug, please check if my proposed fix is the correct one: c8d3d10dc0e90741eed1672adbab2a8a1a1a82bf

Reply:
That looks right to me.